### DISCUSSION TITLE
Could we don't export service_name in OTLP metric exporter?

### DISCUSSION THREAD

**Discussion by aaron-ai** [open-telemetry/opentelemetry-dotnet#4227] [Q&A]
```csharp
var meterProvider = Sdk.CreateMeterProviderBuilder().SetResourceBuilder(ResourceBuilder.CreateEmpty()).build();
```
version: 1.3.1

I set empty resource here, but it doesn't work...

**Comment by cijothomas** [comment]
It looks like you are setting empty Resource explicitly, and if you do that, the resource will be empty.
If you do not  explicitly set anything, it'll do service.name by default.

**Reply by aaron-ai** [reply]
@cijothomas Even if I set an empty resource, the `service_name` will still be exported. In fact, I don't want `service_name` to be exported as a metrics label. How can I achieve this?

**Reply by cijothomas** [reply]
_Even if I set an empty resource, the service_name will still be exported_

Thats not expected. Modified the below program to do `.SetResourceBuilder(ResourceBuilder.CreateEmpty())` and it exports no resource.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/getting-started/Program.cs

**Reply by aaron-ai** [reply]
@cijothomas I could not find the method of `AddConsoleExporter`. I used OtlpExporter for an instead and here is the example:
```csharp
var meterProvider = Sdk.CreateMeterProviderBuilder()
    .SetResourceBuilder(ResourceBuilder.CreateEmpty())
    .AddMeter(MeterName)
    .AddOtlpExporter(delegate(OtlpExporterOptions options, MetricReaderOptions readerOptions)
    {
        options.Protocol = OtlpExportProtocol.Grpc;
        options.Endpoint = new Uri(metric.Endpoints.GrpcTarget);
        options.TimeoutMilliseconds = (int)_client.GetClientConfig().RequestTimeout.TotalMilliseconds;
        options.HttpClientFactory = () => _httpClient;
        readerOptions.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds =
            3 * 1000;
    })
    .Build();
```

but the `service_name` will still be exported.

**Reply by aaron-ai** [reply]
I introduced the dependencies of OpenTelemtry by the command below:

```
dotnet add package OpenTelemetry --version 1.4.0
dotnet add package OpenTelemetry.Api --version 1.4.0
```
